lib/rules/scrollable-region-focusable-matches.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Note:
+ * `excludeHidden=true` for this rule, thus considering only elements in the accessibility tree.
+ */
+const { isHidden, querySelectorAll } = axe.utils;
+const nodeAndDescendents = querySelectorAll(virtualNode, '*');
+
+const hasVisibleChildren = nodeAndDescendents.filter(
```

Comment:
Use `nodeAndDescendents.some()`, that way you're not calling `isVisible` a million times.